tests/networkpolicy_test.go

```diff
@@ -100,13 +100,13 @@ var _ = Describe("Networkpolicy", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It(" should be failed to reach vmia from vmib", func() {
+		It("[test_id:CNV-1511] should be failed to reach vmia from vmib", func() {
```

Comment:
i think some tests should be marked with posneg:negative
no?

Reply:
I think they are not negative test scenarios, since when we created networkpolicy in the namespace, the traffic will be limited by the rules. The failed connections here are only to validate whether the rule is taking effect.

Reply:
makes sense! thanks